vagrant/forum/forum.py

```diff
@@ -40,68 +38,26 @@
 
 # HTML template for an individual comment
 POST = '''\
-    <div class=post><em class=date>%(time)s</em><br>%(content)s</div>
+    <div class=post><em class=date>%s</em><br>%s</div>
```

Comment:
There were %'s in a bunch of the original videos that didn't get redone. The same warnings apply to using {} formats though.